CheckDrive.Web/CheckDrive.Web/Controllers/AccountsController.cs

```diff
@@ -110,7 +110,9 @@ public async Task<IActionResult> Create([Bind("Login,Password,PhoneNumber,FirstN
         public async Task<IActionResult> Edit(int id)
         {
             var account = await _accountDataStore.GetAccountAsync(id);
-
+            var roles = await GETRoles();
```

Comment:
I think, the **GETRoles** method should be changed to **GetRolesAsync** as it is an asynchronous method